Tests/OptimizelyTests-Common/BatchEventBuilderTests_Attributes.swift

```diff
@@ -195,13 +195,16 @@ class BatchEventBuilderTests_Attributes: XCTestCase {
         
         let attributes: [String: Any] = [:]
         
-        _ = try! optimizely.activate(experimentKey: experimentKey,
+        _ = try? optimizely.activate(experimentKey: experimentKey,
                                      userId: userId,
                                      attributes: attributes)
 
-        let json = getFirstEventJSON()!
-        let array = (json["visitors"] as! Array<Dictionary<String, Any>>)[0]["attributes"] as! Array<Dictionary<String, Any>>
-        XCTAssert(array.count == 0)
+        if let json = getFirstEventJSON() {
+            let array = (json["visitors"] as! Array<Dictionary<String, Any>>)[0]["attributes"] as! Array<Dictionary<String, Any>>
+            XCTAssert(array.count == 0)
+        } else {
+            XCTAssert(false)
```

Comment:
Any particular reason for using `XCTAssert(false)` instead of `XCTFail()`

Reply:
I forgot about it :) I'll fix it to XCTFail.